src/analytics/index.ts

```diff
@@ -0,0 +1,82 @@
+import { Identify, identify, init, track } from '@amplitude/analytics-browser'
+import { isProductionEnv } from '../utils/env'
+
+/**
+ * Initializes Amplitude with API key for project.
+ *
+ * Uniswap has two Amplitude projects: test and production. You must be a
+ * member of the organization on Amplitude to view details.
+ */
+export function initializeAnalytics(apiKey: string, proxyUrl: string) {
+  init(
+    apiKey,
+    /* userId= */ undefined, // User ID should be undefined to let Amplitude default to Device ID
+    /* options= */
+    {
+      // Configure the SDK to work with alternate endpoint
+      serverUrl: proxyUrl,
+      // Disable tracking of private user information by Amplitude
+      trackingOptions: {
+        // IP is being dropped before ingestion on Amplitude side, only being used to determine country.
```

Comment:
we can delete this comment, no longer relevant since we aren't sending in IP at all anymore.